src/views/states/list/StatesList.tsx

```diff
@@ -26,6 +26,7 @@ import useDrawerInterface from './hooks/useDrawerInterface';
 import useSelectedFilters from './hooks/useSelectedFilters';
 import useStateColumns from './hooks/useStateColumns';
 import useStateFilters from './hooks/useStateFilters';
+import { FILTERS_TYPES } from './constants';
```

Comment:
nitpick: `FILTERS_TYPES` isn't quite right since each filter type is one type of filter, not many. I think it should be `FILTER_TYPES`.